draft-wang-ppm-differential-privacy.md

```diff
@@ -515,9 +515,30 @@ class DpPolicy:
         pass
 ~~~
 
-## Concrete Instantiations of DP Policies with VDAFs
+## Executing DP Policies in DAP {#dp-in-dap}
 
-### Prio3SumVec, `bits = 1`, with Randomized Response Client-DP
+> TODO: Specify integration of a `DpPolicy` into DAP.
+
+# Use Cases
+
+## Histograms
+
+Many applications require aggregating histograms in which each Client submits a
+bit vector with exactly one bit set. We describe two policies for this use
+case: one which targets the OAMC trust model and provides optimal utility; and
```

Comment:
I don't think readers will be convinced on optimal utility if we don't provide some numbers, but I'm fine with leaving the text as-is, and we will work on it later on.

Reply:
Good call, I've added an author note with this suggestion. (Author note: `[JC: ...]`)